tools/generator/src/DTO/Platform.swift

```diff
@@ -19,14 +19,16 @@ struct Platform: Equatable, Hashable, Decodable {
     struct OSVersion: RawRepresentable, Hashable, Decodable {
         let rawValue: String
         let fullVersion: String
+        let semanticVersion: SemanticVersion?
 
         init?(rawValue: String) {
             self.rawValue = rawValue
 
             var components = rawValue.split(separator: ".")
             let componentCount = components.count
             guard componentCount < 3 else {
-                self.fullVersion = rawValue
+                fullVersion = rawValue
+                semanticVersion = nil
```

Comment:
This should still be set. This guard is for when we don't need to adjust `fullVersion`.

Reply:
Fixed